src/components/table-transactions/table-transactions.scss

```diff
@@ -1,12 +1,22 @@
-.table-transactions{
+.table-transactions {
   border: 1px solid #ddd;
   border-collapse: collapse;
-  width: 100%;
+  width:100%;
+  margin:1%;
   min-width: 600px;
   text-align: left;
   box-sizing: border-box;
-  font-family: $font-table-transactions;
+  font-family: $base-font-family;
   color: #8c8c8c;
+  -moz-box-shadow: 0px 10px 10px -10px #888,
+                   10px 0px 10px -11px #888,
+                   -10px 0px 10px -11px #888;
+  -webkit-box-shadow: 0px 10px 10px -10px #888,
+                      10px 0px 10px -11px #888,
+                      -10px 0px 10px -11px #888;
+  box-shadow: 0px 10px 10px -10px #888,
+              10px 0px 10px -11px #888,
+              -10px 0px 10px -11px #888;
 
```

Comment:
You can do this with MDC Elevation. just add a class mdc-elevation--z2 and import as you did with button
https://material-components-web.appspot.com/elevation.html
and here are principles of choosing shadow
https://material.io/guidelines/material-design/elevation-shadows.html#elevation-shadows-elevation-android